src/main/java/ch/kalunight/zoe/repositories/BannedAccountRepository.java

```diff
@@ -0,0 +1,65 @@
+package ch.kalunight.zoe.repositories;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import ch.kalunight.zoe.model.dto.DTO;
+import net.rithms.riot.constant.Platform;
+
+public class BannedAccountRepository {
+
+  private static final String INSERT_BANNED_ACCOUNT = "INSERT INTO banned_account " +
+      "(banAcc_summonerId, banAcc_server) " +
+      "VALUES ('%s', '%s')";
+  
+  private static final String SELECT_BANNED_ACCOUNT_WITH_SUMMONER_ID_AND_SERVER = "SELECT " + 
+      "banned_account.banacc_id, " + 
+      "banned_account.banacc_summonerid, " + 
+      "banned_account.banacc_server " + 
+      "FROM banned_account " + 
+      "WHERE banned_account.banacc_summonerid = '%s' " + 
+      "AND banned_account.banacc_server = '%s'";
+  
+  private static final String DELETE_BANNED_ACCOUNT_WITH_ID = "DELETE FROM banned_account WHERE banAcc_id = %d";
+  
+  private BannedAccountRepository() {
+    // hide default public constructor
+  }
+  
+  public static void createBannedAccount(String summonerId, Platform platform) throws SQLException {
+    try (Connection conn = RepoRessources.getConnection();
+        Statement query = conn.createStatement();) {
+
+      String finalQuery = String.format(INSERT_BANNED_ACCOUNT, summonerId, platform.getName());
+      query.execute(finalQuery);
+    }
+  }
+  
+  public static DTO.BannedAccount getBannedAccount(String summonerId, Platform platform) throws SQLException {
+    ResultSet result = null;
```

Comment:
Codacy found an issue: [Always check the return of one of the navigation method (next,previous,first,last) of a ResultSet.](https://app.codacy.com/manual/KaluNight/Zoe-Discord-Bot/pullRequest?prid=5905431)